backends/postgres/postgres_backend.go

```diff
@@ -649,7 +649,7 @@ func (p *PgBackend) announceJob(ctx context.Context, queue, jobID string) {
 	defer func(ctx context.Context) { _ = tx.Rollback(ctx) }(ctx)
 
 	// notify listeners that a job is ready to run
-	_, err = tx.Exec(ctx, fmt.Sprintf("NOTIFY %s, '%s'", queue, jobID))
+	_, err = tx.Exec(ctx, fmt.Sprintf(`SELECT pg_notify('%s', '%s')`, queue, jobID))
```

Comment:
On Gitter we were discussing double quoting queue names, although I'm not sure if there's a difference between single and double quotes here.

Reply:
There isn't for pg_notify, since it only takes a `text` type input it does not matter. If we leave it as `NOTIFY ...` then the quoting will matter there. If you prefer the `NOTIFY` I can change that. But I felt it would be best to keep how notifications are kicked off as consistent since we use `pg_notify` in the trigger.